mesh/mesh.hpp

```diff
@@ -858,14 +858,20 @@ class Mesh
        integration rule. */
    /** If the device MemoryType parameter @a d_mt is specified, then the
        returned object will use that type unless it was previously allocated
-       with a different type. */
+       with a different type. The IntegrationRule used with GetGeometricFactors
+       needs to remain valid until the internally stored GeometricFactors
+       objects are destroyed (by either calling Mesh::DeleteGeometricFactors or
+       the Mesh destructor).*/
```

Comment:
I'm not too familiar with doxygen--Will this block comment appear [here](http://mfem.github.io/doxygen/html/classmfem_1_1Mesh.html#aea9acc7eead91d5bd1ac08b9cc23f6bf), or would a user have to actually look at the sourcefile itself to see it?

Reply:
I think I've improved the formatting, it should be more like you want it now.

Reply:
That doxygen page you linked to shows the documentation for the latest release, so it will appear there once we release.  We don't currently maintain a doxygen page for master so if you want the latest you'll have to build the documentation yourself with 'make doc' and find it in the 'doc' folder.